inst/templates/config-template.txt

```diff
@@ -74,4 +75,3 @@ profiles: {{ profiles }}
 # sandpaper and varnish versions) should live
 
 {{{ custom_items }}}
-
```

Comment:
programs typically complain if there is no newline at the end of the file.

Reply:
Good catch. Something I need to adjust in my editor settings, by the look of it.